src/components/button-toggle/button-toggle.spec.js

```diff
@@ -48,13 +48,13 @@ describe('ButtonToggle', () => {
   describe('Modern themes', () => {
     it('renders correctly with small theme', () => {
```

Comment:
Instead of changing that I have replaced it with a `describe.each` with table of themes, as we should not directly rely on theme names in our tests. I must have missed that one.

Reply:
https://github.com/Sage/carbon/commit/7794b8a#diff-c6ab1538324243ee9dfbbe8c4fe1b291 looks good, thanks :white_check_mark: